### ISSUE TITLE
Errors in Mon Lib

### ISSUE THREAD

**Issue by GaribMurshudov** [MRC-LMB-ComputationalStructuralBiology/monomers#25]
There are number of problems in the monomer library identified by Keitaro's tests. We need to address them.
I could see one of the main sources of the problem is torsion angle names for DNA/RNA like molecules. They need to be removed. 
Another source of the problem is links

**Comment by keitaroyam** [comment]
The recent test result: https://github.com/MonomerLibrary/monomers/actions/runs/4680900895/jobs/8292833519
They should be solved if we do what we discussed in https://github.com/MonomerLibrary/monomers/pull/2#issuecomment-1441642883

**Comment by GaribMurshudov** [comment]
We either divide into groups and sort out them. Or I can sort out them. Then it will take some time.

**Comment by keitaroyam** [comment]
One problem is how to automatically create generic links such as TRANS and AA-RNA. Reducing duplicated modifications should be also preferred.
Another problem is links for metal containing compounds.

**Comment by GaribMurshudov** [comment]
Ok. Then we need to think and discuss first. Why do we need to create TRANS links? Don't we have them? 
Rob's accummulator meant to deal with duplicates. It seems that we will need to do some manual work for now. 
Better option could be to use different files for each link with their modification inside that link descriptions. gemmi could be modified to deal with this. However, we should also think about coot.
Links for metal containing ligands will need to be done manually for now. It is a long standing problem. Simple links for metals do not make much sense. However, simple links to non-metal atom of metal containing ligand could be done.

**Comment by GaribMurshudov** [comment]
Ok. I want to try to fix some of the errors in the monomer library. One question: we wanted to have the list of instructions for link generation. i could not find such list in the monomer library. Do we have such list somewhere?

**Comment by keitaroyam** [comment]
I don't think it exists. Some recent ones should be available in PR comments. I think I can try to write a script that generates instructions from mon_lib_list.

**Comment by GaribMurshudov** [comment]
It would be very good to generate a script. Then we can update all but ligands with metals.

**Comment by GaribMurshudov** [comment]
We need a proper script for updating links. As I see many of the errors are related with the fact that monomers have been updated. Links must be updated accordingly. We need to think about this.
MET-TYR related link error: In this link SD of MET becomes +1. In the current link there are additional H atoms that are removed. I have regenarated this link. 
One of the errors is related to SF3 that has Fe in it. It is a bit strange compound and these links are not sufficient. 
It reminds me that we need to find better way of defining metal related links. Simple two residue link is suboptimal
